awswrangler/s3/_merge_upsert_table.py

```diff
@@ -0,0 +1,75 @@
+"""Amazon Merge Upsert Module which can upsert existing table (PRIVATE)."""
+
+import logging
+from typing import List
+
+import pandas
+
+import awswrangler as wr
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
You can remove the first three lines here. Only the `_logger: logging.Logger = logging.getLogger(__name__)` is necessary.

Reply:
Done